lib/api/2.0/notification.js

```diff
@@ -12,6 +12,31 @@ var notificationPost = controller({success: 201}, function(req) {
     return notificationApiService.postNotification(message);
 });
 
+/**
+ * @api {get} /api/2.0/notification/:taskId/:steps
+ * @apiDescription notification get api, used to notify RackHD ipxe initrd downloading finished
+ *  :taskId: active taskId
+ *  :steps: task steps count
+ * @apiName notification-get
+ * @apiGroup notification
+ */
+var notificationIdGet = controller(function(req, res){
+    return notificationApiService.postNotification({
+        taskId: req.swagger.params.taskId.value,
+        progress: {
+            //percentage: "20%",
+            totalSteps: _.parseInt(req.swagger.params.steps.value) || 2,
+            currentStep: 2,
```

Comment:
The same magic number used another time, suggest to define it as a variable or move to constant service.